openmmtools/mcmc.py

```diff
@@ -572,6 +572,9 @@ class BaseIntegratorMove(object):
     context_cache : openmmtools.cache.ContextCache, optional
         The ContextCache to use for Context creation. If None, the global cache
         openmmtools.cache.global_context_cache is used (default is None).
+    reassign_velocities : bool, optional
+        If True, the velocities will be reassigned from the Maxwell-Boltzmann
+        distribution at the beginning of the move (default is False).
```

Comment:
Fair point. The main two cases I had in mind here were in a sequence of moves (in case you wanted to use the final velocities of the previous moves) or for MC integrators that don't sample velocities. The original implementation of `GHMCMove` also didn't reassign velocities so I didn't modify it. In practice, we probably won't use this very often.